src/CodeCompliance/Domain/Checks/PrivateApi/Extended/Extended.php

```diff
@@ -35,6 +36,14 @@ public function getGuideline(): string
         return 'Please avoid extension of the PrivateApi %s in %s';
     }
 
+    /**
+     * @return string
+     */
+    public function getDocumentationUrl(): string
+    {
+        return static::DOCUMENTATION_BASE_URL . 'private-api-method-is-overridden-on-the-project-level.html';
```

Comment:
looks like the correct link
https://docs.spryker.com/docs/scos/dev/guidelines/keeping-a-project-upgradable/upgradability-guidelines/private-api-is-extended.html#example-of-code-that-causes-an-upgradability-error-extending-a-private-api-form-class

Reply:
I am not sure in this case. The link for `PrivateApi:Extension` on this page: https://docs.spryker.com/docs/scos/dev/guidelines/keeping-a-project-upgradable/upgradability-guidelines/upgradability-guidelines.html

goes on the page that I use in code. If you think, that it makes more sense to use another page, I think we need to change it also in current documentation. I can put it also to the new ticket https://spryker.atlassian.net/browse/PPLUS-1626